index.js

```diff
@@ -121,6 +121,7 @@ function provider ({ Biome, blocks, blocksByStateId, toolMultipliers, shapes, ma
   }
 
   Block.prototype.canHarvest = function (heldItemType) {
+    if (!this.harvestTools) { return true }; // for blocks harvestable by hand
     return heldItemType && this.harvestTools && this.harvestTools[heldItemType]
```

Comment:
Could be simplified into
```js
return !this.harvestTools || (heldItemType && this.harvestTools[heldItemType])
```

Reply:
Was just wondering why my bot took so long to mine test-wool